<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseLevelWidget</class>
 <widget class="QWidget" name="ChooseLevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-22</x>
     <y>0</y>
     <width>825</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/resources/jspvz/chooselevelbk.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>392</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第一关</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第二关</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第三关</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第四关</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第五关</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第六关</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第七关</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第八关</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第九关</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_10">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第十关</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第十一关</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_12">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="text">
        <string>第十二关</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关卡选择</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
